AUTHORS

```diff
@@ -0,0 +1,121 @@
+Anastasios Papagiannis <anastasios.papagiannis@isovalent.com>
+Andrei Fedotov <anfedotoff@yandex-team.ru>
+anjmao <andzej.maciusovic@gmail.com>
+Anna Artemova <annartmva@gmail.com>
+Anna Kapuscinska <anna@isovalent.com>
+aohoyd <gh@aohoy.dev>
+arthur-zhang <happyzhangya@gmail.com>
+Ashish Kurmi <akurmi@stepsecurity.io>
+Ashish Naware <ashishnaware3@gmail.com>
+Barun Debnath <barundebnath91@gmail.com>
+Bill Mulligan <billmulligan516@gmail.com>
+Canux <canuxcheng@gmail.com>
+carolina valencia <krol3@users.noreply.github.com>
+Chaiyapruek Muangsiri <cmp.poon@gmail.com>
+Chance Zibolski <chance.zibolski@gmail.com>
+Cheithanya PR <cheithanya2002@gmail.com>
+chenlitw <chenli3792@gmail.com>
+christian2 <christian2@univie.ac.at>
+Christopher Paplham <36825497+cpaplham@users.noreply.github.com>
+darox <maderdario@gmail.com>
+David Windsor <dawindso@cisco.com>
+Dax McDonald <31839142+daxmc99@users.noreply.github.com>
+Dean <22192242+saintdle@users.noreply.github.com>
+dechengyuan <dechengyuan@tencent.com>
+Djalal Harouni <tixxdz@gmail.com>
+Dmitry Savintsev <dmitris@users.noreply.github.com>
+Dmitry Savintsev <dsavints@gmail.com>
+Doğukan ÇELİK <dogukan.celik@itu.edu.tr>
+ekoops <leonardodigiovanna1@gmail.com>
+Eng Zer Jun <engzerjun@gmail.com>
+Feroz Salam <feroz.salam@isovalent.com>
+Filip Nikolic <oss.filipn@gmail.com>
+Frederic Giloux <frederic.giloux@isovalent.com>
+Guozhen She <guozhen.she@snowflake.com>
+hacktivist123 <akintayoshedrack@gmail.com>
+Haiyu Zuo <958474674@qq.com>
+HIHIA <283304489@qq.com>
+hungran <26101787+hungran@users.noreply.github.com>
+Huweicai <i@huweicai.com>
+Ioannis Androulidakis <androulidakis.ioannis@gmail.com>
+Jack-R-lantern <tjdfkr2421@gmail.com>
+James Strong <james.strong@isovalent.com>
+janvi01 <janvibajo1@gmail.com>
+Jianlin Lv <jianlv@ebay.com>
+Jinna C <jinnatim@gmail.com>
+Jiri Olsa <jolsa@kernel.org>
+Joe Stringer <joe@cilium.io>
+John Fastabend <john.fastabend@gmail.com>
+Josh Biggley <jbiggley@users.noreply.github.com>
+Joshua Jorel Lee <joshua.jorel.lee@gmail.com>
+Julien Acroute <julien.acroute@camptocamp.com>
+Justin Chen <mail@justin0u0.com>
+Kevin Conner <kev.conner@getupcloud.com>
+Kevin Sheldrake <kevin.sheldrake@isovalent.com>
+Kohei Morita <mrtc0@ssrf.in>
+Kornilios Kourtis <kornilios@isovalent.com>
+Lancelot <1984737645@qq.com>
+Liang Deng <283304489@qq.com>
+Liz Rice <liz@lizrice.com>
+Lorenz Bauer <lmb@isovalent.com>
+Lucas Fernando Cardoso Nunes <lucasfc.nunes@gmail.com>
+Luigi De Matteis <eilidh3773@gmail.com>
+Mahe Tardy <mahe.tardy@isovalent.com>
+Marwan Nour <38836519+MarwanNour@users.noreply.github.com>
+Matteo Golinelli <34921879+Golim@users.noreply.github.com>
+Mauricio Vásquez <mauriciov@microsoft.com>
+Michael Friedrich <mfriedrich@gitlab.com>
+michael h <mhyatt8008 at g mail com>
+Michael Hyatt <mhyatt8080@gmail.com>
+michalzarsm <michalzarsm@gmail.com>
+Michi Mutsuzaki <michi@isovalent.com>
+mozillazg <mozillazg101@gmail.com>
+mtardy <mahe5397@hotmail.fr>
```

Comment:
```suggestion
```
This is also me! You can remove this line, I merged two commits before joining Isovalent with this email and name.

Reply:
For the other entry, I merged most my commits with my personal address in there but putting the isovalent address in this file makes sense.

Reply:
OK, removed the suggested line.